exercise4/sumOfArray.js

```diff
@@ -0,0 +1,5 @@
+function sumOfArray(array, index = 0) {
+  return index === array.length
```

Comment:
Do you mind modifying this function? I feel there is no need to passing `index` as a parameter. Its better if you just check if the `array.length === 0`.

Reply:
May you check the whole implementation of the code because am not checking if the length of the array is zero